site/schemas/consequence.schema.ts

```diff
@@ -110,3 +107,20 @@ export const consequenceSchema = z.discriminatedUnion("consequenceType", [
 ]);
 
 export type Consequence = z.infer<typeof consequenceSchema>;
+
+export const operatorSchema = z.object({
+    id: z.number(),
+    nocCode: z.string(),
+    operatorPublicName: z.string(),
+    vosaPsvLicenseName: z.string().optional(),
```

Comment:
will we need all of these attributes?

Reply:
Yeah I'm not sure - the API call will be changing early next week, so I figured I'd add it all and we can cut it back as needed.

Reply:
ok that is fine for now

Reply:
Its not about whats being displayed - we only need the NOC and the Operator Name (operatorPublicName) - its about what we need in the back end for when we're using the operator information.